### DISCUSSION TITLE
About definition

### DISCUSSION THREAD

**Discussion by fix-all** [eclipse-ditto/ditto#1926] [General]
hi,
In the "definition" of the Thing I defined, there is a "rel": "tm:extends" that references another model. Then, I used the following request to retrieve the complete definition of this Thing. However, the "required" attribute does not include the "required" attribute of the referenced model.
```curl
curl -X 'GET' \
  'http://127.0.0.1:8080/api/2/things/{thingId}' \
  -H 'accept: application/td+json'
```
Should the "required" attribute of "tm:extends" be included?

**Comment by thjaeckle** [comment]
Could you please share the models you used?
It is difficult without an example to reproduce what you describe ..
